storage/ProcessDatabaseSchema.ts

```diff
@@ -190,20 +208,32 @@ type ProcessDatabaseSchema = NamedSchema<
         pipOrDLA: string;
         whoPIP: string[];
         whoDLA: string[];
-        notes: string;
+        notes: {
+          value: string;
+          isPostVisitAction?: boolean;
+        };
       };
     };
 
     supportNeeds: {
       key: ProcessRef;
       value: {
-        residentSustainmentNotes: string;
-        befriendingNotes: string;
-        adultSafeguardingNotes: string;
-        childrenYoungPeopleSafeguardingNotes: string;
-        domesticSexualViolenceNotes: string;
-        mentalHealth18To65Notes: string;
-        mentalHealthOver65Notes: string;
+        residentSustainmentNotes: {
+          value: string;
+          isPostVisitAction?: boolean;
+        };
+        befriendingNotes: { value: string; isPostVisitAction?: boolean };
```

Comment:
I think I might need to add a migration for these as well, as they're not caught by the current migration that looks for just `notes`

Reply:
Yes. Maybe specify an array of keys to migrate and iterate through them?

Reply:
I'd give them the same type as the other notes values, too.